arch/x86_64/kernel/kernel.cpp

```diff
@@ -1,65 +1,43 @@
-#include <cstdlib/stdio.h>
 #include <stl/array.h>
-#include <x86_64/libk++/ios.h>
+#include <stl/cstdlib/stdio.h>
 #include <x86_64/libk++/iostream.h>
 
 #include <x86_64/drivers/ps2.hpp>
 #include <x86_64/drivers/serial.hpp>
-// #include <x86_64/drivers/vbe.hpp>
-#include <x86_64/drivers/vga.hpp>
-#include <x86_64/init/init.hpp>
-#include <x86_64/int/interrupt.hpp>
-#include <x86_64/multiboot2.hpp>
+#include <x86_64/drivers/vbe.hpp>
+#include <x86_64/kernel.hpp>
+#include <x86_64/trace/strace.hpp>
+
+
 
 [[maybe_unused]] constexpr short MAJOR_VERSION = 0;
 [[maybe_unused]] constexpr short MINOR_VERSION = 0;
 constexpr const char *VERSION_STRING = "0.0";
 
 namespace firefly::kernel::main {
-firefly::drivers::vga::cursor cout;
-
-firefly::drivers::vga::cursor &internal_cursor_handle() {
-    return cout;
-}
-
 void write_ff_info() {
-    using firefly::drivers::vga::clear;
-    clear();
-    firefly::kernel::main::cout << "FireflyOS\nVersion: " << VERSION_STRING << "\nContributors:";
+    printf("FireflyOS\nVersion: %s\nContributors:", VERSION_STRING);
 
     firefly::std::array<const char *, 3> arr = {
         "Lime\t  ", "JohnkaS", "V01D-NULL"
     };
 
     for (size_t i = 0; i < arr.max_size(); i++) {
         if (i % 2 == 0) {
-            firefly::kernel::main::cout << "\n\t";
+            puts("\n\t");
         }
-        firefly::kernel::main::cout << arr[i] << "  ";
+        printf("%s  ", arr[i]);
     }
-    firefly::kernel::main::cout << "\n";
+    puts("\n");

```

Comment:
why are we C-ifying the kernel

Reply:
Because I said I didn't know how to implement << operators with our vbe driver and you were okay with it.
We only had that for vga which I nuked

Reply:
noo not just this, i meant for other things too, dont worry :)